openshift_images/mongodb.adoc

```diff
@@ -10,61 +10,190 @@
 toc::[]
 
 == Overview
-The OpenShift https://github.com/openshift/mongodb/tree/master[MongoDB] image repository contains the sources and Dockerfiles for building various versions of MongoDB images.
+The OpenShift https://github.com/openshift/mongodb/tree/master[MongoDB] image
+repository contains the sources and Dockerfiles for building various versions of
+MongoDB images.
 
 == Versions
 Currently, OpenShift only supports version https://github.com/openshift/mongodb/tree/master/2.4[2.4] of MongoDB.
 
 == Base Images
 
+The following base images are currently available:
+
 * RHEL-7
 * CentOS-7
 
 [NOTE]
 ====
-In order to perform build or test action on a RHEL-7 based MongoDB image, you need to run the build on properly subscribed RHEL-7 machine.
+In order to perform build or test action on a RHEL-7 based MongoDB image, you
+need to run the build on properly subscribed RHEL-7 machine.
 ====
 
 == Installation
-To build MongoDB image, select either a RHEL7 or CentOS7 base image:
+To build a MongoDB image, use one of the following procedures, depending on your
+base image:
+
+*RHEL-7 Base Image*
+
+To build a RHEL-7 based MongoDB image:
```

Comment:
maybe add some notice, similar to the one from the `Test` section:

`To build a RHEL-7-based MongoDB image, run the build on a properly subscribed RHEL machine.`